common/scripted_effects/cwp_political_setup.txt

```diff
@@ -37,6 +37,8 @@ setup_starting_ideologies = {
 					or = {
 						has_law = law_type:law_free_market
 						has_law = law_type:law_interventionism
+						has_law = law_type:law_decentralization
+						has_law = law_type:law_anti_industrialization
 					}
```

Comment:
```suggestion
not = {
  has_law = law_type:law_socialization
}
```


If the only law not here is socialization, it's more efficient to do this instead. less steps for the computer to iterate over.